docs/docsource/reference/prometheus/index.rst

```diff
@@ -0,0 +1,146 @@
+
+  - **toLocationType:** This identifies the type of location to which the information
+    in “toLocation” is targeted. This follows the convention described in 
+    **fromLocation**
+
+  - **status:** This reports the status of the finished task: “success”
+    if the replication completed successfully or “error” if it did not.
+
+  - **contentLengthRange:** This range separates the metric values in different
+    content-length (object size) buckets to offer more meaningful elapsed 
+    time metrics grouped by object size ranges. The labels used are: “<10KB”,
+    “10..30KB”, 30..100KB”, […], “300GB..1TB”, “>1TB”.
+
+Kafka Metrics
+~~~~~~~~~~~~~
+
+Kafka provides its own set of Prometheus metrics, one of which is
+useful for monitoring lifecycle transitions:
+
+**kafka_consumergroup_lag**
+
+  - type: counter
+
+  - description: Lag of consumer groups on a topic/partition,
+    i.e. number of messages published but not consumed yet by this
+    consumer group
+
+  - labels: topic, partition, consumergroup
+
+  The following labels can be used to filter this metric:
+
+  - **topic:** The topic name. The relevant topic names for transition policies
+    are:
+
+    - “backbeat-data-mover” for tracking the data mover (replication) topic
+
+    - “backbeat-lifecycle-bucket-tasks” for tracking the bucket tasks
+      topic (scanning buckets for lifecycle actions to execute on objects)
+
+    - “backbeat-lifecycle-object-tasks” for tracking the object tasks
+      topic (tasks to execute on each expiration or a transition object)
+
+    - “backbeat-gc” for tracking the garbage collection topic (The garbage 
+      collection service removes original data after a transition. It is also
+      used for transient source data removal after successful CRR.)
+
+  - **partition:** Partition number in the Kafka topic
+
+  - **consumergroup:** This label carries the consumer group name. Relevant 
+    consumer group names for transition policies are:
+
+    - backbeat-replication-group-[location name]: Consumer group
+      that consumes replication tasks of transition actions to a
+      particular location (e.g. “backbeat-replication-group-aws1”
+      tracks the consumer group for the "aws1" location). Consumer groups
+      will also consume messages for other locations, but will not process 
+      them (hence the lag will also count replications to other locations).
+
+    - backbeat-lifecycle-bucket-processor-group: The unique consumer
+      group of the “backbeat-lifecycle-bucket-tasks” topic.
+
+    - backbeat-lifecycle-object-processor-group: The unique consumer
+      group of the “backbeat-lifecycle-object-tasks” topic.
+
+    - backbeat-gc-consumer-group: The unique consumer group of the
+      “backbeat-gc” topic
+
+You can use either the Prometheus API or the Prometheus UI and the PromQL 
+query language to access these metrics.
+
+Below is an few example. See the Prometheus documentation for a full PromQL
```

Comment:
As discussed offline, you might say for now "Below is one example".

Reply:
Your diplomatic papers are on their way, sir!